From andrew@lunn.ch  Fri Dec 18 10:28:39 2009
From: Andrew Lunn <andrew@lunn.ch>
Date: Sat, 12 Dec 2009 23:39:46 +0100
Subject: Staging: batman-adv: Fix the naming of the debug option.
To: gregkh@suse.de
Cc: devel@linuxdriverproject.org, Andrew Lunn <andrew@lunn.ch>, Marek Lindner <lindner_marek@yahoo.de>
Message-ID: <1260657587-1135-6-git-send-email-andrew@lunn.ch>


So that the configuration hierarchy is correct, set the debug option
to have the same base as the main BATMAN option.

Signed-off-by: Andrew Lunn <andrew@lunn.ch>
Signed-off-by: Marek Lindner <lindner_marek@yahoo.de>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/batman-adv/Kconfig |    4 ++--
 drivers/staging/batman-adv/README  |    2 +-
 drivers/staging/batman-adv/main.c  |    2 +-
 drivers/staging/batman-adv/main.h  |    4 ++--
 4 files changed, 6 insertions(+), 6 deletions(-)

--- a/drivers/staging/batman-adv/Kconfig
+++ b/drivers/staging/batman-adv/Kconfig
@@ -14,10 +14,10 @@ config BATMAN_ADV
         http://www.open-mesh.org/ for more information and user space
         tools.
 
-config BATMAN_DEBUG
+config BATMAN_ADV_DEBUG
 	bool "B.A.T.M.A.N. debugging"
 	depends on BATMAN_ADV != n
-	help
+	---help---
 
 	  This is an option for use by developers; most people should
 	  say N here. This enables compilation of support for
--- a/drivers/staging/batman-adv/main.c
+++ b/drivers/staging/batman-adv/main.c
@@ -57,7 +57,7 @@ atomic_t module_state;
 
 struct workqueue_struct *bat_event_workqueue;
 
-#ifdef CONFIG_BATMAN_DEBUG
+#ifdef CONFIG_BATMAN_ADV_DEBUG
 int debug;
 
 module_param(debug, int, 0644);
--- a/drivers/staging/batman-adv/main.h
+++ b/drivers/staging/batman-adv/main.h
@@ -76,7 +76,7 @@
 			 * broadcasting / etc */
 #define DBG_ROUTES 2	/* route or hna added / changed / deleted */
 
-#ifdef CONFIG_BATMAN_DEBUG
+#ifdef CONFIG_BATMAN_ADV_DEBUG
 extern int debug;
 
 extern int bat_debug_type(int type);
@@ -85,7 +85,7 @@ extern int bat_debug_type(int type);
 			printk(KERN_DEBUG "batman-adv:" fmt, ## arg);	\
 	}								\
 	while (0)
-#else /* !CONFIG_BATMAN_DEBUG */
+#else /* !CONFIG_BATMAN_ADV_DEBUG */
 #define bat_dbg(type, fmt, arg...) do {		\
 	}					\
 	while (0)
--- a/drivers/staging/batman-adv/README
+++ b/drivers/staging/batman-adv/README
@@ -115,7 +115,7 @@ kernel, use "make menuconfig" and enable
 "B.A.T.M.A.N. debugging". When compiling outside of the kernel tree it
 is necessary to edit the file Makefile.kbuild and uncomment the line
 
-#EXTRA_CFLAGS += -DCONFIG_BATMAN_DEBUG
+#EXTRA_CFLAGS += -DCONFIG_BATMAN_ADV_DEBUG
 
 The additional debug output is by default disabled. It can be enabled
 either at kernel modules load time or during run time. To enable debug
